src/contributing_to_atmosphere/contributing_to_atmosphere.md

```diff
@@ -14,6 +15,55 @@ Atmosphere or 'atmo' can refer to several different things:
 
 For clarity, this page will disambiguate all uses of "Atmosphere".
 
+## What are the steps to contributing to Atmosphere/Core services?
+
+Contribution to core-services products (usually) takes place in two different environments:
+
+1. [JIRA](https://pods.iplantcollaborative.org/jira/secure/Dashboard.jspa) - This is an external service where we document bugs, issues, and new features to be fixed.
```

Comment:
@steve-gregory I'm not sure what you mean by "This is an external service". We're writing for an audience who does not have access to JIRA, right?